transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebAcceptProcessor.java

```diff
@@ -100,7 +100,7 @@ protected void flushInternal(final WsebSession session) {
         }
 
         // get write request queue and process it
-        final WriteRequestQueue writeRequestQueue = session.getWriteRequestQueue();
+        final WriteRequestQueue writeRequestQueue = session.getTransportSession().getWriteRequestQueue();
```

Comment:
Shouldn't this be processor for TransportSession as it is primarily flushing TransportSession messages ?